src/foam/nanos/alarming/services.jrl

```diff
@@ -41,6 +40,7 @@ p({
   "serve":true,
   "serviceScript":`
     return new foam.dao.EasyDAO.Builder(x)
+      .setLastModifiedAware(true)
```

Comment:
might not be needed, by default easyDAO should add the decorator if model implements the interface.